stacks-signer/src/signer.rs

```diff
@@ -409,6 +409,39 @@ impl Signer {
         self.coordinator_selector.last_message_time = Some(Instant::now());
     }
 
+    /// Resend outbound messages if no state change has occurred within the configured timeout
+    pub fn resend_outbound_messages(&mut self, response_wait_timeout: Duration) {
+        // We do not need to resend outbound messages if we are in the Idle state.
+        if self.state == State::Idle {
+            return;
+        }
+        match self.signer_db.outbound_messages_lookup(self.reward_cycle) {
+            Ok(Some(info)) => {
+                if info.insertion_time < chrono::Utc::now() - response_wait_timeout
```

Comment:
For these kinds of things, it's better to use `Duration` because a change to the system clock won't affect the delay (and this can and does happen regularly if the user is running an NTP daemon).

Reply:
Oh nm, I see -- this time has to also be loaded and stored into the DB, and this isn't possible with `Duration`.  What you have is fine.